src/usr/local/www/system_certmanager.php

```diff
@@ -464,6 +478,21 @@
 				break;
 			case 'import':
 				/* Import an external certificate+key */
+				if ($pkcs12_data) {
+					$pconfig['cert'] = $pkcs12_data['cert'];
+					$pconfig['key'] = $pkcs12_data['pkey'];
+					if ($_POST['pkcs12_intermediate'] && is_array($pkcs12_data['extracerts'])) {
+						foreach ($pkcs12_data['extracerts'] as $intermediate) {
+							$int_data = openssl_x509_parse($intermediate);
+							if (!$int_data) continue;
```

Comment:
The structure of this `if` needs fixed. It needs braces and should be on multiple lines.

Reply:
fixed